eng/tools/TypeSpecValidation/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
```

Comment:
@mikeharder  is a nest tsconfig needed or would the one in the root work for thes?

Reply:
there might be some issues with the `outDir` and `include` resolution

Reply:
I was able to import the root `tsconfig.json` and remove some of the settings, but I do think this project needs a few customizations.  It might be possible to minimize this further but it was the best I could do in a few minutes.